server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -202,7 +202,7 @@ public enum MetadataVersion {
     IBP_3_7_IV4(19, "3.7", "IV4", false),
 
     // New version for the Kafka 3.8.0 release.
-    IBP_3_8_IV0(20, "3.8", "IV0", true),
+    IBP_3_8_IV0(20, "3.8", "IV0", false),
```

Comment:
I believe this was an oversight while doing the revert. 3.8-IV0 should have no metadata changes

Reply:
done